titus_isolate/allocate/forecast_ip_cpu_allocator.py

```diff
@@ -285,19 +281,24 @@ def __compute_new_placement(
             self.__call_meta['ip_solver_call_args']['use_per_workload'] = use_per_workload
 
         try:
+            placement_solver = PlacementSolver(
+                total_available_cus=num_threads,
+                num_sockets=num_packages,
+                solver_params=self.__ip_solver_params)
+
             start_time = time.time()
-            placement, status = optimize_ip(
-                requested_units,
-                num_threads,
-                num_packages,
+
+            placement, status, _, _ = placement_solver.optimize(
```

Comment:
@rostyboost : Do we want the other return values for anything now?

Reply:
no